.travis.yml

```diff
@@ -5,38 +5,38 @@ env:
     global:
         - use_codecov=true
     matrix:
-        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
+        - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
```

Comment:
Can you use a global variable to avoid the need to re-specify this all over id we change this in the future?

Reply:
Currently we have 5 times the CentOS image and 7 times the Fedora image which is fairly even and it might be possible that either of these images might be used more in the future (we might move unit tests to CentOS or more integration tests to Fedora or remove the Python 3.7 unit test completely). I am not sure that we will certainly save something.

Reply:
I meant only the change portions, i.e. `docker.io`.